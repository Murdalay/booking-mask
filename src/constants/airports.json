[
  {
    "code": "OCH",
    "name": "A. L. Mangham Jr. Regional Airport",
    "location": "Nacogdoches, Texas, United States"
  },
  {
    "code": "APH",
    "name": "A.P. Hill Army Airfield",
    "location": "Fort A.P. Hill, Virginia, United States"
  },
  { "code": "AAL", "name": "Aalborg Airport", "location": "Aalborg, Denmark" },
  {
    "code": "AAP",
    "name": "Aappilattoq (Qaasuitsup) Heliport",
    "location": "Aappilattoq, Greenland"
  },
  { "code": "AAR", "name": "Aarhus Airport", "location": "Aarhus, Denmark" },
  { "code": "JEG", "name": "Aasiaat Airport", "location": "Aasiaat, Greenland" },
  { "code": "JIM", "name": "Aba Segud Airport", "location": "Jimma, Ethiopia" },
  {
    "code": "DIR",
    "name": "Aba Tenna Dejazmach Yilma International Airport",
    "location": "Dire Dawa, Ethiopia"
  },
  { "code": "ABD", "name": "Abadan Airport", "location": "Abadan, Iran" },

  { "code": "BGH", "name": "Abbaye Airport", "location": "Boghe (Bogu\u00e9), Mauritania" },
  { "code": "XAB", "name": "Abbeville Aerodrome", "location": "Abbeville, France" },
  {
    "code": "YXX",
    "name": "Abbotsford Airport",
    "location": "Abbotsford, British Columbia, Canada"
  },
  { "code": "AAW", "name": "Abbottabad Airport", "location": "Abbottabad, Pakistan" },
  { "code": "EAB", "name": "Abbse Airport", "location": "Abbse, Yemen" },
  {
    "code": "MLG",
    "name": "Abdul Rachman Saleh Airport",
    "location": "Malang, East Java, Indonesia"
  },
  { "code": "SNU", "name": "Abel Santa Mar\u00eda Airport", "location": "Santa Clara, Cuba, Cuba" },
  { "code": "AEA", "name": "Abemama Atoll Airport", "location": "Abemama Atoll, Kiribati" },
  { "code": "ABZ", "name": "Aberdeen Airport", "location": "Aberdeen, Scotland, United Kingdom" },
  {
    "code": "ABR",
    "name": "Aberdeen Regional Airport",
    "location": "Aberdeen, South Dakota, United States"
  },
  { "code": "GZS", "name": "Abernathy Field", "location": "Pulaski, Tennessee, United States" },
  {
    "code": "AHB",
    "name": "Abha Regional Airport",
    "location": "Abha / Khamis Mushait, Saudi Arabia"
  },
  {
    "code": "ABI",
    "name": "Abilene Regional Airport",
    "location": "Abilene, Texas, United States"
  },
  { "code": "ABG", "name": "Abingdon Airport", "location": "Abingdon, Queensland, Australia" },
  { "code": "ABO", "name": "Aboisso Airport", "location": "Aboisso, C\u00f4te d'Ivoire" },
  { "code": "AOD", "name": "Abou-Deia Airport", "location": "Abou-Deia, Chad" },
  {
    "code": "CJS",
    "name": "Abraham Gonz\u00e1lez International Airport",
    "location": "Ciudad Ju\u00e1rez, Chihuahua, Mexico"
  },
  {
    "code": "SPI",
    "name": "Abraham Lincoln Capital Airport",
    "location": "Springfield, Illinois, United States"
  },
  {
    "code": "AJS",
    "name": "Abreojos Airport",
    "location": "Abreojos, Baja California Sur, Mexico"
  },
  {
    "code": "AUH",
    "name": "Abu Dhabi International Airport",
    "location": "Abu Dhabi, United Arab Emirates"
  },
  { "code": "AUE", "name": "Abu Rudeis Airport", "location": "Abu Rudeis, Egypt" },
  { "code": "ABS", "name": "Abu Simbel Airport", "location": "Abu Simbel, Egypt" },
  { "code": "AEH", "name": "Ab\u00e9ch\u00e9 Airport", "location": "Ab\u00e9ch\u00e9, Chad" },
  {
    "code": "ARA",
    "name": "Acadiana Regional Airport",
    "location": "New Iberia, Louisiana, United States"
  },
  { "code": "ARF", "name": "Acaricuara Airport", "location": "Acaricuara, Colombia" },
  {
    "code": "MFV",
    "name": "Accomack County Airport",
    "location": "Melfa, Virginia, United States"
  },
  { "code": "ACS", "name": "Achinsk Airport", "location": "Achinsk, Krasnoyarski, Russia" },
  { "code": "SRG", "name": "Achmad Yani International Airport", "location": "Semarang, Indonesia" },
  { "code": "ACU", "name": "Achutupo Airport", "location": "Achutupo, Panama" },
  { "code": "AAD", "name": "Ad-Dabbah Airport", "location": "Ad-Dabbah, Sudan" },
  {
    "code": "ADT",
    "name": "Ada Municipal Airport (FAA: ADH)",
    "location": "Ada, Oklahoma, United States"
  },
  {
    "code": "ADK",
    "name": "Adak Island Airport",
    "location": "Adak Island, Alaska, United States"
  },
  { "code": "ADA", "name": "Adana Sakirpasa Airport", "location": "Adana, Turkey" },
  {
    "code": "EKX",
    "name": "Addington Field (Elizabethtown Regional Airport)",
    "location": "Elizabethtown, Kentucky, United States"
  },
  { "code": "ADS", "name": "Addison Airport", "location": "Dallas, Texas, United States" },
  {
    "code": "HUU",
    "name": "Alf\u00e9rez FAP David Figueroa Fernandini Airport",
    "location": "Hu\u00e1nuco, Hu\u00e1nuco Region, Peru"
  },
  {
    "code": "AXG",
    "name": "Algona Municipal Airport (FAA: AXA)",
    "location": "Algona, Iowa, United States"
  },
  { "code": "KEZ", "name": "Ali Al Salem Airport", "location": "Ali Al Salem, Kuwait" },
  { "code": "AII", "name": "Ali-Sabieh Airport", "location": "Ali-Sabieh, Djibouti" },
  {
    "code": "ALI",
    "name": "Alice International Airport",
    "location": "Alice, Texas, United States"
  },
  {
    "code": "ASP",
    "name": "Alice Springs Airport",
    "location": "Alice Springs, Northern Territory, Australia"
  },
  { "code": "ALZ", "name": "Alitak Seaplane Base", "location": "Lazy Bay, Alaska, United States" },
  { "code": "AJO", "name": "Aljouf Airport", "location": "Aljouf, Yemen" },
  { "code": "LON", "name": "All Airports", "location": "London, United Kingdom" },
  { "code": "STO", "name": "All Airports", "location": "Stockholm, Sweden" },
  {
    "code": "QBA",
    "name": "All Airports",
    "location": "San Francisco Bay Area, California, United States"
  },
  { "code": "TYO", "name": "All Airports", "location": "Tokyo, Japan" },
  { "code": "QHO", "name": "All Airports", "location": "Houston, Texas, United States" },
  {
    "code": "QSF",
    "name": "All Airports",
    "location": "San Francisco Bay Area, California, United States"
  },
  {
    "code": "SEL",
    "name": "All Airports (code was used in Gimpo International Airport)",
    "location": "Seoul, South Korea"
  },
  {
    "code": "OSA",
    "name": "All Airports (code was used in Osaka International Airport)",
    "location": "Osaka, Japan"
  },
  { "code": "AOE", "name": "Anadolu Airport", "location": "Eskisehir, Turkey" },
  {
    "code": "AKP",
    "name": "Anaktuvuk Pass Airport",
    "location": "Anaktuvuk Pass, Alaska, United States"
  },
  { "code": "QNB", "name": "Anand Airport", "location": "Anand, India" },
  { "code": "ANS", "name": "Andahuaylas Airport", "location": "Andahuaylas, Peru" },
  { "code": "ADC", "name": "Andakombe Airport", "location": "Andakombe, Papua New Guinea" },
  {
    "code": "ADO",
    "name": "Andamooka Airport",
    "location": "Andamooka, South Australia, Australia"
  },
  { "code": "UAM", "name": "Andersen Air Force Base", "location": "Agana, Guam" },
  {
    "code": "AID",
    "name": "Anderson Municipal Airport (Darlington Field)",
    "location": "Anderson, Indiana, United States"
  },
  {
    "code": "AND",
    "name": "Anderson Regional Airport",
    "location": "Anderson, South Carolina, United States"
  },
  { "code": "ADN", "name": "Andes Airport", "location": "Andes, Colombia" },
  { "code": "AZN", "name": "Andizhan Airport", "location": "Andizhan, Uzbekistan" },
  { "code": "ALV", "name": "Andorra La Vella Airport", "location": "Andorra la Vella, Andorra" },
  { "code": "ADV", "name": "Andover Airport", "location": "Andover, England, United Kingdom" },
  {
    "code": "ADW",
    "name": "Andrews Air Force Base",
    "location": "Camp Springs, Maryland, United States"
  },
  {
    "code": "RHP",
    "name": "Andrews-Murphy Airport",
    "location": "Andrews, North Carolina, United States"
  },
  { "code": "WAD", "name": "Andriamena Airport", "location": "Andriamena, Madagascar" },
  {
    "code": "ASD",
    "name": "Andros Town International Airport",
    "location": "Andros Island, Bahamas"
  },
  { "code": "ANL", "name": "Andulo Airport", "location": "Andulo, Angola" },
  { "code": "ANX", "name": "And\u00f8ya Airport, Andenes", "location": "Andenes, Norway" },
  {
    "code": "ARE",
    "name": "Antonio (Nery) Juarbe Pol Airport (FAA: ABO)",
    "location": "Arecibo, Puerto Rico"
  },
  {
    "code": "GUM",
    "name": "Antonio B. Won Pat International Airport (Guam International Airport)",
    "location": "Tamuning, Guam (near Hag\u00e5t\u00f1a), United States"
  },
  { "code": "CUM", "name": "Antonio Jos\u00e9 de Sucre Airport", "location": "Cumana, Venezuela" },
  { "code": "SCU", "name": "Antonio Maceo International Airport", "location": "Santiago, Cuba" },
  { "code": "PSO", "name": "Antonio Nari\u00f1o Airport", "location": "Pasto, Colombia" },
  { "code": "VQS", "name": "Antonio Rivera Rodriguez Airport", "location": "Vieques, Puerto Rico" },
  {
    "code": "APO",
    "name": "Antonio Rold\u00e1n Betancourt Airport",
    "location": "Apartad\u00f3, Colombia"
  },
  {
    "code": "ACB",
    "name": "Antrim County Airport",
    "location": "Bellaire, Michigan, United States"
  },
  { "code": "ANM", "name": "Antsirabato Airport", "location": "Antalaha, Madagascar" },
  { "code": "ATJ", "name": "Antsirabe Airport", "location": "Antsirabe, Madagascar" },
  { "code": "ANR", "name": "Antwerp International Airport", "location": "Antwerp, Belgium" },
  {
    "code": "ANH",
    "name": "Anuha Island Resort Airport",
    "location": "Anuha Island Resort, Solomon Islands"
  },
  { "code": "ADP", "name": "Anuradhapura Airport", "location": "Anuradhapura, Sri Lanka" },
  { "code": "ANV", "name": "Anvik Airport", "location": "Anvik, Alaska, United States" },
  {
    "code": "AYN",
    "name": "Anyang Airport",
    "location": "Anyang, Henan, People's Republic of China"
  },
  {
    "code": "APS",
    "name": "An\u00e1polis Airport",
    "location": "An\u00e1polis, Goi\u00e1s, Brazil"
  },
  { "code": "AOJ", "name": "Aomori Airport", "location": "Aomori, Japan" },
  {
    "code": "AAF",
    "name": "Apalachicola Municipal Airport",
    "location": "Apalachicola, Florida, United States"
  },
  { "code": "AAS", "name": "Apalapsili Airport", "location": "Apalapsili, Indonesia" },
  { "code": "QAP", "name": "Apapa Airport", "location": "Apapa, Nigeria" },
  { "code": "APK", "name": "Apataki Airport", "location": "Apataki, French Polynesia" },
  { "code": "APB", "name": "Apolo Airport", "location": "Apolo, Bolivia" },
  {
    "code": "APV",
    "name": "Apple Valley Airport",
    "location": "Apple Valley, California, United States"
  },
  { "code": "APR", "name": "April River Airport", "location": "April River, Papua New Guinea" },
  { "code": "APU", "name": "Apucarana Airport", "location": "Apucarana, Paran\u00e1, Brazil" },
  { "code": "AKX", "name": "Aqtobe Airport", "location": "Aqtobe (Aktyubinsk), Kazakhstan" },
  { "code": "ARW", "name": "Arad International Airport", "location": "Arad, Romania" },
  {
    "code": "ARS",
    "name": "Aragar\u00e7as Airport",
    "location": "Aragar\u00e7as, Goi\u00e1s, Brazil"
  },
  { "code": "ARP", "name": "Aragip Airport", "location": "Aragip, Papua New Guinea" },
  {
    "code": "AUX",
    "name": "Aragua\u00edna Airport",
    "location": "Aragua\u00edna, Tocantins, Brazil"
  },
  { "code": "AXC", "name": "Aramac Airport", "location": "Aramac, Queensland, Australia" },
  {
    "code": "BFE",
    "name": "Bielefeld Airport",
    "location": "Bielefeld, North Rhine-Westphalia, Germany"
  },
  {
    "code": "BBE",
    "name": "Big Bell Airport",
    "location": "Big Bell, Western Australia, Australia"
  },
  { "code": "BGK", "name": "Big Creek Airport", "location": "Big Creek, Belize" },
  { "code": "BGQ", "name": "Big Lake Airport", "location": "Big Lake, Alaska, United States" },
  {
    "code": "BMX",
    "name": "Big Mountain Air Force Station (FAA: 37AK)",
    "location": "Big Mountain, Alaska, United States"
  },
  {
    "code": "BPI",
    "name": "Big Piney-Marbleton Airport",
    "location": "Big Piney, Wyoming, United States"
  },
  {
    "code": "YTL",
    "name": "Big Trout Lake Airport",
    "location": "Big Trout Lake, Ontario, Canada"
  },
  {
    "code": "FOZ",
    "name": "Bigfork Municipal Airport",
    "location": "Bigfork, Minnesota, United States"
  },
  {
    "code": "BIF",
    "name": "Biggs Army Airfield",
    "location": "Fort Bliss, El Paso, Texas, United States"
  },
  { "code": "BBI", "name": "Biju Patnaik Airport", "location": "Bhubaneswar, India" },
  { "code": "BII", "name": "Bikini Atoll Airport", "location": "Bikini Atoll, Marshall Islands" },
  { "code": "PAB", "name": "Bilaspur Airport", "location": "Bilaspur, India" },
  { "code": "BIO", "name": "Bilbao Airport", "location": "Bilbao, Spain" },
  { "code": "BIJ", "name": "Biliau Airport", "location": "Biliau, Papua New Guinea" },
  {
    "code": "BIW",
    "name": "Billiluna Airport",
    "location": "Billiluna, Western Australia, Australia"
  },
  {
    "code": "BIL",
    "name": "Billings Logan International Airport",
    "location": "Billings, Montana, United States"
  },
  { "code": "BLL", "name": "Billund Airport", "location": "Billund, Denmark" },
  {
    "code": "YTZ",
    "name": "Billy Bishop Toronto City Airport",
    "location": "Toronto, Ontario, Canada"
  },
  {
    "code": "HSE",
    "name": "Billy Mitchell Airport",
    "location": "Hatteras, North Carolina, United States"
  },
  { "code": "BMU", "name": "Bima Airport", "location": "Bima, Indonesia" },
  { "code": "BVI", "name": "Birdsville Airport", "location": "Birdsville, Queensland, Australia" },
  { "code": "XBJ", "name": "Birjand Airport", "location": "Birjand, Iran" },
  {
    "code": "BHX",
    "name": "Birmingham International Airport",
    "location": "Birmingham, England, United Kingdom"
  },
  {
    "code": "BHM",
    "name": "Birmingham-Shuttlesworth International Airport",
    "location": "Birmingham, Alabama, United States"
  },
  {
    "code": "SBU",
    "name": "Blue Earth Municipal Airport",
    "location": "Blue Earth, Minnesota, United States"
  },
  { "code": "LEX", "name": "Blue Grass Airport", "location": "Lexington, Kentucky, United States" },
  {
    "code": "MTV",
    "name": "Blue Ridge Airport",
    "location": "Martinsville, Virginia, United States"
  },
  { "code": "BEF", "name": "Bluefields Airport", "location": "Bluefields, Nicaragua" },
  { "code": "BNU", "name": "Blumenau Airport", "location": "Blumenau, Santa Catarina, Brazil" },
  { "code": "BLH", "name": "Blythe Airport", "location": "Blythe, California, United States" },
  {
    "code": "HKA",
    "name": "Blytheville Municipal Airport",
    "location": "Blytheville, Arkansas, United States"
  },
  { "code": "KBS", "name": "Bo Airport", "location": "Bo, Sierra Leone" },
  {
    "code": "BVB",
    "name": "Boa Vista International Airport",
    "location": "Boa Vista, Roraima, Brazil"
  },
  { "code": "BNV", "name": "Boana Airport", "location": "Boana, Papua New Guinea" },
  { "code": "BOV", "name": "Boang Airport", "location": "Boang, Papua New Guinea" },
  {
    "code": "IAN",
    "name": "Bob Baker Memorial Airport",
    "location": "Kiana, Alaska, United States"
  },
  { "code": "BUR", "name": "Bob Hope Airport", "location": "Burbank, California, United States" },
  { "code": "CEW", "name": "Bob Sikes Airport", "location": "Crestview, Florida, United States" },
  {
    "code": "HBG",
    "name": "Bobby L. Chain Municipal Airport",
    "location": "Hattiesburg, Mississippi, United States"
  },
  { "code": "BOY", "name": "Bobo Dioulasso Airport", "location": "Bobo-Dioulasso, Burkina Faso" },
  { "code": "BCT", "name": "Boca Raton Airport", "location": "Boca Raton, Florida, United States" },
  {
    "code": "BOC",
    "name": "Bocas del Toro \"Isla Colon\" International Airport",
    "location": "Bocas Town, Bocas del Toro, Panama"
  },
  { "code": "BNM", "name": "Bodinumu Airport", "location": "Bodinumu, Papua New Guinea" },
  { "code": "BXN", "name": "Bodrum-Imsik Airport", "location": "Bodrum, Turkey" },
  { "code": "BOO", "name": "Bod\u00f8 Airport", "location": "Bod\u00f8, Norway" },
  {
    "code": "BFI",
    "name": "Boeing Field/King County International Airport",
    "location": "Seattle, Washington, United States"
  },
  {
    "code": "BNB",
    "name": "Boende Airport",
    "location": "Boende, Democratic Republic of the Congo"
  },
  { "code": "XBG", "name": "Bogande Airport", "location": "Bogande, Burkina Faso" },
  { "code": "TOF", "name": "Bogashevo Airport", "location": "Tomsk, Russia" },
  {
    "code": "GIC",
    "name": "Boigu Island Airport",
    "location": "Boigu Island, Queensland, Australia"
  },
  { "code": "ASH", "name": "Boire Field", "location": "Nashua, New Hampshire, United States" },
  {
    "code": "BOI",
    "name": "Boise Air Terminal (Gowen Field)",
    "location": "Boise, Idaho, United States"
  },
  {
    "code": "CTQ",
    "name": "Do Palmar Airport",
    "location": "Santa Vit\u00f3ria, Rio Grande do Sul, Brazil"
  },
  { "code": "DOA", "name": "Doany Airport", "location": "Doany, Madagascar" },
  {
    "code": "MGE",
    "name": "Dobbins Air Reserve Base (Atlanta NAS)",
    "location": "Marietta, Georgia, United States"
  },
  { "code": "DOB", "name": "Dobo Airport", "location": "Dobo, Wamar Island, Indonesia" },
  {
    "code": "DKV",
    "name": "Docker River Airport",
    "location": "Docker River, Northern Territory, Australia"
  },
  {
    "code": "CNQ",
    "name": "Doctor Fernando Piragine Niveyro International Airport",
    "location": "Corrientes, Corrientes, Argentina"
  },
  {
    "code": "TOB",
    "name": "Dodge Center Airport",
    "location": "Dodge Center, Minnesota, United States"
  },
  { "code": "TAS", "name": "Yuzhny Airport", "location": "Tashkent, Uzbekistan" },
  { "code": "EYL", "name": "Y\u00e9liman\u00e9 Airport", "location": "Y\u00e9liman\u00e9, Mali" },
  {
    "code": "CRT",
    "name": "Z.M. Jack Stell Field",
    "location": "Crossett, Arkansas, United States"
  },
  { "code": "ACZ", "name": "Zabol Airport", "location": "Zabol, Iran" },
  { "code": "XZA", "name": "Zabr\u00e9 Airport", "location": "Zabr\u00e9, Burkina Faso" },
  { "code": "ZAD", "name": "Zadar Airport", "location": "Zadar, Croatia" },
  { "code": "ZAG", "name": "Zagreb Airport", "location": "Zagreb, Croatia" },
  { "code": "ZAH", "name": "Zahden Airport", "location": "Zahden, Iran" },
  { "code": "AKM", "name": "Zakouma Airport", "location": "Zakouma, Chad" },
  {
    "code": "ZTH",
    "name": "Zakynthos International Airport, \"Dionysios Solomos\"",
    "location": "Zakynthos, Greece"
  },
  { "code": "BBZ", "name": "Zambezi Airport", "location": "Zambezi, Zambia" },
  {
    "code": "ZAM",
    "name": "Zamboanga International Airport",
    "location": "Zamboanga City, Philippines"
  },
  { "code": "TOA", "name": "Zamperini Field", "location": "Torrance, California, United States" },
  { "code": "ANJ", "name": "Zanaga Airport", "location": "Zanaga, Republic of the Congo" },
  {
    "code": "ZZV",
    "name": "Zanesville Municipal Airport",
    "location": "Zanesville, Ohio, United States"
  },
  { "code": "ZNZ", "name": "Zanzibar Airport", "location": "Zanzibar, Tanzania" },
  { "code": "APZ", "name": "Zapala Airport", "location": "Zapala, Neuqu\u00e9n, Argentina" },
  { "code": "AZT", "name": "Zapatoca Airport", "location": "Zapatoca, Colombia" },
  { "code": "ZAZ", "name": "Zaragoza Airport", "location": "Zaragoza, Spain" },
  { "code": "ZAJ", "name": "Zaranj Airport", "location": "Zaranj, Afghanistan" },
  { "code": "DJE", "name": "Zarzis Airport", "location": "Djerba, Tunisia" },
  { "code": "SZI", "name": "Zaysan Airport", "location": "Zaysan, Kazakhstan" },
  {
    "code": "ZPH",
    "name": "Zephyrhills Municipal Airport",
    "location": "Zephyrhills, Florida, United States"
  },
  { "code": "ZER", "name": "Zero Airport", "location": "Ziro, India" },
  {
    "code": "DYG",
    "name": "Zhangjiajie Airport (formerly Dayong Airport)",
    "location": "Zhangjiajie, Hunan, China"
  },
  {
    "code": "ZHA",
    "name": "Zhanjiang Airport",
    "location": "Zhanjiang, Guangdong Province, China"
  },
  {
    "code": "CGO",
    "name": "Zhengzhou Xinzheng International Airport",
    "location": "Zhengzhou, Henan, China"
  },
  { "code": "DZN", "name": "Zhezkazgan Airport", "location": "Zhezkazgan, Kazakhstan" },
  { "code": "PZH", "name": "Zhob Airport", "location": "Zhob, Pakistan" },
  { "code": "DAC", "name": "Zia International Airport", "location": "Dhaka, Bangladesh" },
  { "code": "IEG", "name": "Zielona G\u00f3ra Airport", "location": "Zielona G\u00f3ra, Poland" },
  { "code": "ZIG", "name": "Ziguinchor Airport", "location": "Ziguinchor, Senegal" },
  { "code": "ZND", "name": "Zinder Airport", "location": "Zinder, Niger" },
  {
    "code": "OBL",
    "name": "Zoersel-Oostmalle Airfield",
    "location": "Zoersel / Oostmalle, Belgium"
  },
  { "code": "ORG", "name": "Zorg en Hoop Airport", "location": "Paramaribo, Suriname" },
  { "code": "MCZ", "name": "Zumbi dos Palmares Airport", "location": "Macei\u00f3, Brazil" },
  {
    "code": "ZYI",
    "name": "Zunyi Airport",
    "location": "Zunyi, Guizhou, People's Republic of China"
  },
  { "code": "ZRH", "name": "Zurich International Airport", "location": "Z\u00fcrich, Switzerland" },
  { "code": "EVN", "name": "Zvartnots International Airport", "location": "Yerevan, Armenia" },
  { "code": "ZQW", "name": "Zweibr\u00fccken Airport", "location": "Zweibr\u00fccken, Germany" },
  {
    "code": "MDQ",
    "name": "\u00c1stor Piazzola International Airport",
    "location": "Mar del Plata, Argentina"
  },
  {
    "code": "AGH",
    "name": "\u00c4ngelholm-Helsingborg Airport (former ICAO: ESDB)",
    "location": "\u00c4ngelholm, Sweden"
  },
  { "code": "AES", "name": "\u00c5lesund Airport, Vigra", "location": "Aalesund, Norway" },
  {
    "code": "OSD",
    "name": "\u00c5re \u00d6stersund Airport",
    "location": "\u00d6stersund, Sweden"
  },
  { "code": "CKZ", "name": "\u00c7anakkale Airport", "location": "\u00c7anakkale, Turkey" },
  { "code": "DNZ", "name": "\u00c7ardak Airport", "location": "Denizli, Turkey" },
  { "code": "TEQ", "name": "\u00c7orlu Airport", "location": "Tekirdag, Turkey" },
  {
    "code": "EPL",
    "name": "\u00c9pinal - Mirecourt Airport",
    "location": "Juvaincourt, Vosges, France"
  },
  {
    "code": "IFJ",
    "name": "\u00cdsafj\u00f6r\u00f0ur Airport",
    "location": "\u00cdsafj\u00f6r\u00f0ur, Iceland"
  },
  {
    "code": "BMY",
    "name": "\u00cele Art - Waala Airport (Belep Islands Airport)",
    "location": "Waala, Belep Islands, New Caledonia"
  },
  {
    "code": "YGR",
    "name": "\u00celes-de-la-Madeleine Airport",
    "location": "Iles-de-la-Madeleine, Quebec, Canada"
  },
  {
    "code": "ULG",
    "name": "\u00d6lgii Airport",
    "location": "Bayan-\u00d6lgii Province, Mongolia"
  },
  {
    "code": "UNR",
    "name": "\u00d6nd\u00f6rkhaan Airport",
    "location": "\u00d6nd\u00f6rkhaan, Khentii, Mongolia"
  },
  { "code": "ORB", "name": "\u00d6rebro Airport", "location": "\u00d6rebro, Sweden" },
  {
    "code": "OER",
    "name": "\u00d6rnsk\u00f6ldsvik Airport",
    "location": "\u00d6rnsk\u00f6ldsvik, Sweden"
  },
  {
    "code": "HOV",
    "name": "\u00d8rsta/Volda Airport, Hovden",
    "location": "Hovden, M\u00f8re og Romsdal, Norway"
  },
  {
    "code": "SQQ",
    "name": "\u0160iauliai International Airport",
    "location": "\u0160iauliai, Lithuania"
  },
  { "code": "ZBK", "name": "\u017dabljak Airport", "location": "\u017dabljak, Montenegro" },
  { "code": "ILZ", "name": "\u017dilina Airport", "location": "\u017dilina, Slovakia" }
]
