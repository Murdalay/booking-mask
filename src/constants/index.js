import keyMirror from 'fbjs/lib/keyMirror';

/**
 * @namespace Constants
 * @desc App constants
 */

/**
 * @constant {Object} ActionTypes
 * @memberof Constants
 */
export const ActionTypes = keyMirror({
  EXCEPTION: undefined,
  TOGGLE_ROUND_TRIP: undefined,
  BOOKING_TOGGLE_PASSANGERS_DROPDOWN: undefined,
  BOOKING_CHANGE_PASSANGERS_NUMBER: undefined,
  BOOKING_CHANGE_FROM: undefined,
  BOOKING_CHANGE_TO: undefined,
  BOOKING_CHANGE_FLIGHT_DATES: undefined,
  BOOKING_SUBMIT: undefined,
  BOOKING_SUBMIT_SUCCESS: undefined,
  BOOKING_SET_INVALID: undefined,
  BOOKING_UNSET_INVALID: undefined,
  BOOKING_URL_READY: undefined,
  AIROPORTS_LIST_GET: undefined,
  AIROPORTS_LIST_GET_SUCCESS: undefined,
  AIROPORTS_LIST_GET_FAILURE: undefined,
  SHOW_ALERT: undefined,
  HIDE_ALERT: undefined,
});

/**
 * @constant {Object} STATUS
 * @memberof Constants
 */
export const STATUS = {
  IDLE: 'idle',
  RUNNING: 'running',
  READY: 'ready',
  SUCCESS: 'success',
  ERROR: 'error',
};
