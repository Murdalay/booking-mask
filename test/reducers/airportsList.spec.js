import reducer from 'reducers/airportsList';
import { ActionTypes } from 'constants/index';

describe('Booking', () => {
  it('should return the initial state', () => {
    expect(reducer.airportsList(undefined, {})).toMatchSnapshot();
  });

  it(`should handle ${ActionTypes.AIRPORTS_LIST_GET}`, () => {
    expect(
      reducer.airportsList(undefined, {
        type: ActionTypes.AIRPORTS_LIST_GET,
        payload: {},
      }),
    ).toMatchSnapshot();
  });

  it(`should handle ${ActionTypes.AIRPORTS_LIST_GET_SUCCESS}`, () => {
    expect(
      reducer.airportsList(undefined, {
        type: ActionTypes.AIRPORTS_LIST_GET_SUCCESS,
        payload: { data: ['FOO', 'BAR', 'BAZ'] },
      }),
    ).toMatchSnapshot();
  });

  it(`should handle ${ActionTypes.AIRPORTS_LIST_GET_FAILURE}`, () => {
    expect(
      reducer.airportsList(undefined, {
        type: ActionTypes.AIRPORTS_LIST_GET_FAILURE,
        payload: {},
      }),
    ).toMatchSnapshot();
  });
});
