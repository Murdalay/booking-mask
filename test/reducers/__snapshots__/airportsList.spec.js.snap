// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Booking should handle AIRPORTS_LIST_GET 1`] = `
Object {
  "data": Array [],
  "message": "",
  "status": "running",
}
`;

exports[`Booking should handle AIRPORTS_LIST_GET_FAILURE 1`] = `
Object {
  "data": Array [],
  "message": "Something went wrong",
  "status": "error",
}
`;

exports[`Booking should handle AIRPORTS_LIST_GET_SUCCESS 1`] = `
Object {
  "data": Array [
    "FOO",
    "BAR",
    "BAZ",
  ],
  "message": "",
  "status": "ready",
}
`;

exports[`Booking should return the initial state 1`] = `
Object {
  "data": Array [],
  "message": "",
  "status": "idle",
}
`;
